vuepress/docs/next/tutorials/devops/installation/open-shift/openshift-install-by-operator-hub.md

```diff
@@ -262,7 +262,33 @@ Now we'll create the application itself.
 
 See the [Next Steps](#next-steps) below to continue your work with Entando.
 
+## Appendix A - Troubleshooting and Common Errors
+
+### SSL Certificate Error
```

Comment:
Let's rename this to "Image Pull Error"

Reply:
Technically SSL isn't involved, when the images for the pods are pulled from the registry, some environments will have blocked docker.io and so you'll get errors pulling images. Sorry, had to dig into the details again to realize that. We'll want a similar change on line 268 to match.